cmd/ci-secret-bootstrap/main.go

```diff
@@ -120,6 +122,13 @@ func (o *options) completeOptions(censor *secrets.DynamicCensor, kubeConfigs map
 		return err
 	}
 
+	if vals := o.secretNamesRaw.Strings(); len(vals) > 0 {
+		secretNames := sets.NewString(vals...)
+		logrus.Info("Removing secrets whose name is not config-updater ...")
+		configUpdaterOnly(&o.config, secretNames)
```

Comment:
This method and preceeding log call should also be genericized

Reply:
Updated.